ng-dev/ts-circular-dependencies/index.ts

```diff
@@ -119,6 +107,34 @@ export function main(
     Log.warn(`   Please rerun with "--warnings" to inspect unresolved imports.`);
   }
 
+  if (goldenFile) {
```

Comment:
NIt: I'd prefer if we guard by `approve` at the first condition, as that would make it easier to follow IMO.

(maybe that's just me though)